components/optimizely-context.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
This file just removes the need to have `if (typeof window === "Object")` or whatever. We can now access optimizely's scripts via this.context.optimizely for both server and client.